src/Administration/Resources/app/administration/src/app/assets/scss/global.scss

```diff
@@ -110,3 +110,9 @@ a[target="_blank"]:not(.sw-external-link, .sw-internal-link, .mt-button) {
         min-height: 45px !important;
     }
 }
+
+.mt-url-field {
```

Comment:
Shouldn't we fix this in the component library?

Reply:
Yes, we open an issue, but meanwhile, we fix it here

Reply:
We shouldn't do hotfixes that aren't even marked as such. It's much easier to fix style issues by just updating the component library, rather than having to remove hotfixes again. Especially if it's an in-house library.
If it's that urgent, mention it in the appropriate channels, or open a PR yourself.

Reply:
We are gathering all these little issues and creating the issues in GH to be fixed. Even we fix them directly, right now QA is waiting for all these fixes to have the new release ready ASAP.

I added the link to the GH issue, to remove this code when it is fixed.